data/ka'het/ka'het ships.txt

```diff
@@ -197,7 +203,11 @@ ship "Telis'het" "Telis'het (EMP)"
 		"EMP Torpedo" 64
 
 		"Ka'het Shield Restorer" 11
-		"Ka'het Primary Cooling" 2
+		"Ka'het Primary Cooling"
+		"Ka'het Support Cooling"
+		"Ka'het MHD Generator" 2
+		"Ka'het Reserve Accumulator" 3
+		"Outfits Expansion"
```

Comment:
It seems odd to me that Ka'het ships would be using outfits expansions. Why not just increase the outfit space of the ships to account for the new outfits? Given that these ships can not be captured, it's not as if we need to have the outfit space perfectly balanced.